File: src/main/java/dev/hiitsdevin/pastelwonderland/Tools/PastelTools.java
package dev.hiitsdevin.pastelwonderland.Tools;

import dev.hiitsdevin.pastelwonderland.Base.Items;
import net.minecraft.item.ToolMaterial;
import net.minecraft.recipe.Ingredient;

public class PastelTools implements ToolMaterial {

    public static final PastelTools PASTEL_TOOLS = new PastelTools();

    @Override
    public int getDurability() {
        return 256;
    }

    @Override
    public float getMiningSpeedMultiplier() {
        return 4.0f;
    }

    @Override
    public float getAttackDamage() {
        return 3.0f;
    }

    @Override
    public int getMiningLevel() {
        return 2;
    }

    @Override
    public int getEnchantability() {
        return 9;
    }

    @Override
    public Ingredient getRepairIngredient() {
        return Ingredient.ofItems(Items.PASTEL_INGOT);
    }
}

File: src/main/kotlin/devos/pastelwonderland/Tools/PastelTools.kt
package devos.pastelwonderland.Tools

import devos.pastelwonderland.Base.Items
import net.minecraft.item.ToolMaterial
import net.minecraft.recipe.Ingredient

class PastelTools : ToolMaterial {
    override fun getDurability(): Int {
        return 256
    }

    override fun getMiningSpeedMultiplier(): Float {
        return 4.0f
    }

    override fun getAttackDamage(): Float {
        return 3.0f
    }

    override fun getMiningLevel(): Int {
        return 2
    }

    override fun getEnchantability(): Int {
        return 9
    }

    override fun getRepairIngredient(): Ingredient {
        return Ingredient.ofItems(Items.PASTEL_INGOT)
    }

    companion object {
        val PASTEL_TOOLS = PastelTools()
    }
}